### PR TITLE
add css option - fixes #228

### LINKED ISSUE #228: Add an option for custom CSS
It would be great to have an option to allow custom CSS to be applied on the webpage by phantomJS before rendering the screenshot. Do you plan to add this feature in the future ?

Thx

### PR DESCRIPTION
**Pull request by SamVerschueren**
This PR resolves #228 by bumping `screenshot-stream`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "node": ">=0.12.0"
   },
   "scripts": {
-    "test": "xo && npm run prepublish && nyc ava",
+    "test": "xo && npm run prepublish && nyc ava --serial",
```
Comment (SamVerschueren) [inline package.json]: Had to add `--serial` because if they run concurrently, they did not complete on my system. Maybe something to do with Phantom?

Comment (sindresorhus) [inline package.json]: Maybe, or something with a newer AVA release. Does the tests pass if you set `"screenshot-stream": "3.0.0"` and reinstall dependencies and rerun the tests? I'm on slow 3G today, so can't test.

I don't think making it `--serial` is the correct solution, though.

**Comment by SamVerschueren** [comment]
Removed the `--serial` flag and apparently it runs on Travis :).

**Comment by sindresorhus** [comment]
Thank you. New version published. Would you be up for updating the CLI and grunt task too? No worries if not :)

**Comment by SamVerschueren** [comment]
Sure, no problem :).

**Comment by SamVerschueren** [comment]
Weird, tests failed on `new Map()` in `mem` on node 5 and 4. Any idea except for restarting them?

**Comment by sindresorhus** [comment]
No, but that seemed to do it.

**Pull request closed without merging**